### ISSUE TITLE
how to access map from mobile (android)?

### ISSUE THREAD

**Issue by CharliePrown** [AHAAAAAAA/PokemonGo-Map#1893]
I only know this as desktop version, which tags pokemons as disapperars in....short times always, some minutes.

So its kinda useless to have a look and then go outside.

Any way to use this/access the 5000 on the go from an android device?

thanks

**Comment by mquandalle** [comment]
There is a mobile client in the work at https://github.com/omkarmoghe/Pokemap

Alternatively, if you prefer to use the AHAAAAAAA's version running on your computer, you can expose publicly one port of your internet router and redirect it to your local IP port which run the application—if your ISP provide an option for that.

**Comment by jesusdrodriguez** [comment]
@CharliePrown You can use ngrok to open up a port and forward it to a private website, you will get this private website link address on the ngrok command window once its created. On your phone, you can then visit said given link and you will have access to the forwarded data from your localhost:5000

To do this, follow the next steps:

Download and install ngrok: https://ngrok.com/download on your PC.
Run ngrok.exe and enter "ngrok http 5000" in the command line.
Enter the generated forwarding address in your phone's browser (doesn't matter if its the http or the https link, simply enter said website into your phone's browser).

**Comment by AppliedEllipsis** [comment]
mchristopher/PokemonGo-DesktopMap moved away from ngrok due to rate limits.
They migrated to localtunnel.me.

Fixes some loading bugs with Google Maps, switching to localtunnel.me for mobile since it doesn't have rate limits.

**Comment by jesusdrodriguez** [comment]
@AppliedEllipsis what kind of rate limits, I haven't had any issues so far with ngrok, but I'm only scanning a single city.

**Comment by timikx** [comment]
I also used ngrok and localtunnel.me but now I'm using heroku, you just need to register there and then use "deploy tu heroku" button from here:
https://github.com/AHAAAAAAA/PokemonGo-Map/wiki/Heroku-Deployment
after that you can access it from phone, it is very easy

**Comment by CharliePrown** [comment]
that localtunnel thing is safe for the local machine setup on to hacks?)

**Comment by AppliedEllipsis** [comment]
@jesusdrodriguez I don't know details.  I just know it was in one of the latest commits.

**Comment by CharliePrown** [comment]
Btw, as it is only a website on the local machine, common teamviewer should work fine too?

Connect, open the browser, done.

**Comment by jesusdrodriguez** [comment]
I don't trust teamviewer anymore (they were recently hacked), but sure if you can get that to work.

**Comment by CharliePrown** [comment]
did not know that.

so ngrok would be simple and most safe then, as i can input a user/password

only ngrok itself as creator of the link might compromise then? :dancer: 

thx

**Comment by jesusdrodriguez** [comment]
That's correct, but the thing with ngrok is that you own the link to your website via port forwarding, as long as your ngrok program is running on your laptop/desktop then you will have access to that private "link" that was created for you. As soon as you shut down that program it will be deleted, if you restart the program again it will create a completely different random private "link" so the previous one will never be used again.

For your input of user/password you should be using a dummy/trash/alt account at all times, never use your main account.

**Comment by Jazz23** [comment]
What I use is the [chrome remote desktop](https://chrome.google.com/webstore/detail/chrome-remote-desktop/gbchcmhmhahfdphkhkmpfmihenigjmpp?hl=en) app to control my computer from my phone. It works like a charm!
